R/opq.R

```diff
@@ -18,6 +19,9 @@
 #' \dontrun{
 #' overpass_query(q) -> reading_noms
 #' sp::plot(reading_noms$osm_nodes)
+#' p <- overpass_query(opq("Leeds") %>%
+#'   add_feature("leisure", "park"))
+#' sp::plot(p$osm_polygons[1,])
```

Comment:
This all has to be considered in relation to [this issue](https://github.com/osmdatar/osmdata/issues/1), the only practicable resolution of which is likely to be the necessity of defining a (simple S3) class for the returned object. Defining a class would also allow us to return, for example, the actual query and a timestamp. As for your probs: Inserting items of defined classes into a list is not really good practise (and is my fault here!), and is likely the reason for your plot issues.